patterns-use-cases/ticket-reservation/ticket-reservation-python/example/cart_object.py

```diff
@@ -0,0 +1,58 @@
+from datetime import timedelta
+
+from restate.context import ObjectContext
+from restate.object import VirtualObject
+
+from example.checkout_service import handle
+from example.ticket_object import reserve, mark_as_sold, unreserve
+
+cart = VirtualObject("cart")
+
+
+@cart.handler()
+async def add_ticket(ctx: ObjectContext, ticket_id: str) -> bool:
+    reserved = await ctx.object_call(reserve, key=ticket_id, arg=None)
+
+    if reserved:
+        tickets = await ctx.get("tickets") or []
+        tickets.append(ticket_id)
+        ctx.set("tickets", tickets)
+
+        ctx.object_send(expire_ticket, key=ctx.key(), arg=ticket_id, send_delay=timedelta(minutes=15))
+
+    return reserved
+
+
+@cart.handler()
+async def checkout(ctx: ObjectContext) -> bool:
+    tickets = await ctx.get("tickets") or []
+
+    if len(tickets) == 0:
+        return False
+
+    success = await ctx.service_call(handle, arg={'user_id': ctx.key(),
```

Comment:
To improve readibility, i suggest to do import renaming of other handlers. e.g.

```
from example.checkout_service import handle as checkout_handle
```

And then here:

```suggestion
    success = await ctx.service_call(checkout_handle, arg={'user_id': ctx.key(),
```